screens/AuthScreen.spec.tsx

```diff
@@ -0,0 +1,93 @@
+import '@testing-library/jest-native/extend-expect';
```

Comment:
another new staff :+1: I have never seen before this .spec file :thinking:

Reply:
Are you talking about this import? Then it took me a long time to find how to use it with our react-native-testing-library... but finally it works.

And regarding .spec files, I decided that we can use:
.spec.tsx files for integration tests (react-native-testing-library)
.test.tsx files for unit tests (with enzyme)
In general it's only a convention.